### PR TITLE
docs: Computer report 

### PR DESCRIPTION
**Pull request by yash1412**
# Checklist

- [ ] fixes #xxxx
- [x] Passes `Semantic Pull Request` and `TypoCheck`
- [ ] New docs are listed in `README.md`
- [x] New docs are listed in `index.md`
- [ ] Updated table of content
- [ ] Updated nav links in mkdocs.yml

---

## PR description

Write a brief note here what does your PR changes.

### CODE REVIEW

**Comment by pnijhara** [comment]
It's the third time you are making same mistake with "ROS Engineer" spelling.

File: docs/computer/student_Reports/yash_pranshi_saloni_2020.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+
+| Title   |      Author     |  Email-Id  |
+|:----------:|:-------------:|:------:|
+| Report for the tech stack learned till now at AUV club, ZHCET | Yash Agarwal | yashagarwal1214@gmail.com |
```
Comment (syedjawadakhtar) [inline docs/computer/student_Reports/yash_pranshi_saloni_2020.md]: Do not make a table. The format is to be in YAML.
```
---
title:
author:
email-id:
---
```

File: docs/computer/student_Reports/yash_pranshi_saloni_2020.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+# Linux Command Interface
+
+**Objective**: To get comfortable with the Linux Command Line.
+
+**Introduction**: The Command Line Interface (CLI), is a non-graphical, text-based interface to the computer system, where the user types in a command and the computer then successfully executes it. The terminal is the platform or the IDE that provides the command line interface (CLI) environment to the user. The CLI terminal accepts the commands that the user types and passes to a shell. The shell then receives and interprets what the user has typed into the instructions that can be executed by the OS (Operating System)
+
+**Equipments/Software used**: Linux OS.
+
+**Procedure**: Followed some tutorials. [Reference](https://www.youtube.com/playlist?list=PLS1QulWo1RIb9WVQGJ_vh-RQusbZgO_As)
```
Comment (syedjawadakhtar) [inline docs/computer/student_Reports/yash_pranshi_saloni_2020.md]: While writing procedure... don't be ambigious by writing 'some'. Be straight forward

File: docs/computer/student_Reports/yash_pranshi_saloni_2020.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+**Results**: Completed
+
+**Future work**: To contribute in major open source projects.
+
+# Computer Vision(OpenCv)
```
Comment (syedjawadakhtar) [inline docs/computer/student_Reports/yash_pranshi_saloni_2020.md:43]: If this is a different task write it in a different report with this title

File: docs/electronics/README.md [hunk 4]
```diff
@@ -10,6 +10,7 @@ Follow markdown format and files name ending with .md. For adding images create
 
 ## Resources
 
+<<<<<<< HEAD
```
Comment (syedjawadakhtar) [inline docs/electronics/README.md]: You didn't remove the conflict line headers properly.

**Review by syedjawadakhtar — CHANGES_REQUESTED** [review]
Proof Read at least once and then send the PR

File: docs/computer/student_Reports/yash_pranshi_saloni_2020.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
+---
+Title: Report for the tech stack learned till now at AUV club, ZHCET
```
Comment (pnijhara) [inline docs/computer/student_Reports/yash_pranshi_saloni_2020.md]: You have made them go uppercase. Write the same way as Jawad suggested.

Comment (pnijhara) [reply docs/computer/student_Reports/yash_pranshi_saloni_2020.md]: Make it title, author and email-id.

**Review by pnijhara — CHANGES_REQUESTED** [review]

File: docs/computer/student_Reports/yash_pranshi_saloni_2020.md (hunk 5 above)
Comment (yash1412) [reply docs/computer/student_Reports/yash_pranshi_saloni_2020.md]: ok

**Review by pnijhara — APPROVED** [review]

**Pull request merged**